lua/toggleterm/terminal.lua

```diff
@@ -360,6 +360,19 @@ local function opener(size, term)
   end
 end
 
+---Spawn terminal background job in a buffer without a window
+function Terminal:spawn()
+  if fn.bufexists(self.bufnr) == 0 then
```

Comment:
is this equivalent to `api.nvim_buf_is_valid(self.bufnr)` if so it would be good to avoid the `lua->viml` round trip via `vim.fn`

Reply:
I was using the same condition as in https://github.com/akinsho/toggleterm.nvim/blob/6c7f5dbdd69bc5611a85194ddca83ac2c8ee84d6/lua/toggleterm/terminal.lua#L373 but I can change both if you want.